### ISSUE TITLE
Option without modify source code

### ISSUE THREAD

**Issue by paulocoutinhox** [rouault/pdfium_build_gdal_3_4#1]
Hi,

There is any option without change pdfium source code?

I already have the binaries here:
https://github.com/paulo-coutinho/pdfium-lib

I want only link with it.

There is any way to do without modify pdfium?

**Comment by rouault** [comment]
> There is any way to do without modify pdfium?

No, the GDAL PDF driver needs patches to have new capabilities and we haven't figured it yet how to contribute them to upstream pdfium and if there would be welcome by them
